src/__tests__/testParsing.py

```diff
@@ -398,6 +424,23 @@ def testFileMatch(self):
             self.checkFileResult(testCase)
         print('Tested %d cases.' % len(fileTestCases))
 
+    def testAllInputMatches(self):
+        for testCase in allInputTestCases:
+            result = parse.matchLine(testCase['input'], False, True)
+
+            if not result:
+                self.assertTrue(testCase['match'] is None,
+                                'Expected a match "%s" where one did not occur.' %
+                                testCase['match'])
+
```

Comment:
nit: rm this line for readability, since its hard to tell what block continue is for